peachjam/views/courts.py

```diff
@@ -108,3 +116,58 @@ def get_context_data(self, **kwargs):
 
 class CourtYearView(CourtDetailView):
     queryset = Judgment.objects.prefetch_related("judges")
+
+
+class CourtRegistryDetailView(CourtDetailView):
+    queryset = Judgment.objects.prefetch_related("judges")
+    template_name = "peachjam/court_registry_detail.html"
+
+    def get_base_queryset(self):
+        qs = super().get_base_queryset().filter(registry=self.registry)
+        if "year" in self.kwargs:
+            qs = qs.filter(date__year=self.kwargs["year"])
+        return qs
+
+    def get_queryset(self):
+        self.court = get_object_or_404(Court, code=self.kwargs["code"])
+        self.registry = get_object_or_404(
+            CourtRegistry, code=self.kwargs["registry_code"]
+        )
+        return super().get_queryset()
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["registry"] = self.registry
+        context["formatted_registry_name"] = self.registry.name
```

Comment:
If you keep this as `formatted_court_name`, then it'll get used in the appropriate place in the template without you having to change anything. This is the reason we have `formatted_court_name` -- so that it can have extra data added if necessary, like the year or a registry.